dist/prow_rustfmt.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Install and run cargofmt. Input: rust version
+test -z ${1} || rustup default ${1}
```

Comment:
I suggest to use the `--toolchain` argument to `rustup` instead of setting a new default. That way it can also be used on developer machines.

Reply:
Good idea, implemented